### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by davotoula** [vitorpamplona/amethyst#943]
I'm not an expert on software licenses but this seems to be code from somewhere else?

https://github.com/vitorpamplona/amethyst/blob/683af698c664f5590224db5419ac3b0a88d7ba1e/amethyst/src/main/java/androidx/compose/material3/pullrefresh/PullRefresh.kt#L2C4-L2C13

**Comment by vitorpamplona** [comment]
Yep, I think Google already offers a pull to refresh now. Google had deprecated one of their libraries and copying the code was the official temporary answer. 😳

**Comment by davotoula** [comment]
Should I look into whether we can remove that copy paste code?

Or just deleting the license text?

**Comment by davotoula** [comment]
seems to be removed
